CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+ComplianceAsCode Contribution Guidelines
+===
+
+We're so glad you're thinking about contributing to ComplianceAsCode! If you're unsure about anything, just ask -- or submit the issue or pull request to start the conversation! The worst that could happen is you'll politely be asked to change something.
+
+We want to ensure a welcoming environment for all members of the ComplianceAsCode community. Reading the projects CONTRIBUTING policy (you are here), its [LICENSE](https://github.com/ComplianceAsCode/content/blob/master/LICENSE), and its [README](https://github.com/ComplianceAsCode/content/blob/master/README.md), will help you get calibrated.
+
+This project is a public-private partnership with the United States Government as part of its [open source initiatives](https://code.nsa.gov/). The project has welcomed contributions from across the United States Government, private industry, and countries worldwide. Currently, there are four project maintainers who are accountable to ensure a welcoming community and our code of conduct is followed. Should you ever need, please feel free to reach out directly!
```

Comment:
With stressing "public-private partnership with the United States Government" in the contribution guidelines you bring implications about some administrative or supervisory role of it in the project. Which means that there are certain constraints for contributions to the project. It would be great to briefly describe these constraints and (or) give a more direct link (referring to https://code.nsa.gov/ is way too vague).